internal/app/server/handler_test.go

```diff
@@ -88,6 +88,105 @@ func TestNewParcel(t *testing.T) {
 	}
 }
 
+func TestGetParcel(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	parcel := model.Parcel{
+		UserID:             1,
+		SourceAddress:      "Dhaka Bangladesh",
+		DestinationAddress: "Pabna Shadar",
+		SourceTime:         time.Date(2020, time.April, 11, 21, 34, 01, 0, time.UTC),
+		ParcelType:         "Document",
+		Price:              200.0,
+		CarrierFee:         180.0,
+		CompanyFee:         20.0,
+		CreatedAt:          time.Date(2020, time.April, 11, 21, 34, 01, 0, time.UTC),
+		UpdatedAt:          time.Date(2020, time.April, 11, 21, 34, 01, 0, time.UTC),
+	}
+
+	testCases := []struct {
+		desc          string
+		mockParcelSvc func() *mocks.MockParcelService
+		parcelID      string
+		expStatusCode int
+		expResponse   string
+	}{
+		{
+			desc: "should success",
+			mockParcelSvc: func() *mocks.MockParcelService {
+				s := mocks.NewMockParcelService(ctrl)
+				s.EXPECT().GetParcelByID(gomock.Any(), 1).Return(parcel, nil)
```

Comment:
use parcelID instead of 1. It will be good for me to understand var name instead of value.